docs/streams/upgrade-guide.html

```diff
@@ -64,12 +70,21 @@ <h1>Upgrade Guide and API Changes</h1>
         See <a href="#streams_api_changes_0101">below</a> a complete list of 0.10.1 API changes that allow you to advance your application and/or simplify your code base, including the usage of new features.
     </p>
 
+    <!-- TODO: verify release verion and update `id` and `href` attributes (also at other places that link to this headline) -->
+    <h3><a id="streams_api_changes_120" href="#streams_api_changes_120">Streams API changes in 1.2.0</a></h3>
+    <p>
+      Kafka 1.2.0 allows to manipulate timestamps of output records using the Processor API (<a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-251%3A+Allow+timestamp+manipulation+in+Processor+API">KIP-251</a>).
+      To enable this new feature, <code>ProcessorContext#forward(...)</code> was modified.
+      The two existing overloads <code>#forward(Object key, Object value, String childName)</code> and <code>#forward(Object key, Object value, int childIndex)</code> were deprecated and a new overload <code>#forward(Object key, Object value, To to)</code> was added.
+      The new class <code>To</code> allows to send record to all or specific downstream processors by name and to set the timestamp for the output record.
```

Comment:
This doesn't quite sound right..
Maybe "The new class <code>To</code> allows you to send records..."